hardware/detect.py

```diff
@@ -861,6 +862,25 @@ def detect_rtc_clock(hw_lst):
     hw_lst.append(('system', 'rtc', 'utc', rtc.get_rtc()))
 
 
+def detect_auxv(hw_lst):
+    new_env = os.environ.copy()
+    new_env["LD_SHOW_AUXV"] = "1"
+
+    uuid_cmd = Popen("/bin/true",
+                     env=new_env,
+                     stdout=PIPE)
+    stdout, err = uuid_cmd.communicate()
+    if err is not None:
```

Comment:
Can we have something printed to stderr as a breadcrumb

Reply:
Done